docs/emasser/index.md

```diff
@@ -189,11 +202,11 @@ Each CLI endpoint command have several layers of help.
     Retrieve artifacts for a system or system/filename combination
 
     USAGE
-      $ saf emasser get artifacts [ARGUMENTS]
+      $ saf emasser get artifacts [ARGUMENT]
```

Comment:
Good point, here is the rationality:
**ARGUMENT**
We use ARGUMENT when the command is of the form (using the artifacts endpoint): `saf emasser get artifacts [forSystem or export]`, per oclif these are arguments are simple parameters that are given on the system's command line

If you invoke this command `saf emasser get artifacts -h` you get a list of arguments used with the command

We use the uppercase here because that is how oclif list them.

**options**
Flag options are non-positional arguments passed to the command, if a command is invoke with the -h flag (aka option) the list of options that are available are listed, some are requrired, other are optional, hence the word options.

I hope that this clarifies things.